scripts/ExtractShotData.jl

```diff
@@ -1,41 +1,88 @@
+#!/usr/bin/env julia
+
 include("../data-utils/Utils.jl")
+
 import PrettyPrint
 import JSON
+import CSV
+import DataFrames
+
+output_filename_stem = "data/shot_data"
 
 function usage()
-    println("Usage: julia <optionss> Utils.jl <data directory> <output filename>")
+    println("USAGE: julia <options> $(PROGRAM_FILE) <output_filename>")
 end
 
 "Trim down the shot data to only the parameters we're interested in for now."
 function trim_shot_data(events)
     retval = []
     for event in events
-        tdict = Dict()
-        tdict["location"] = event["location"]
-        tdict["body_part"] = event["shot"]["body_part"]["name"]
-        tdict["outcome"] = event["shot"]["outcome"]["name"]
-        tdict["type"] = event["shot"]["type"]["name"]
+        location = event["location"]
```

Comment:
I actually don't want to filter at this stage, but this works for now. If we're gonna do this, would rather we compared lowercase(type) to lowercase("Open Play")